classroomio/classroomio
# style: dark mode support

## What does this PR do?

Adds dark mode support for classriomio.com with default to system preference.

## Type of change


## How should this be tested?



- Manual test
Navigates through all the pages either by having your system in dark mode and expecting the applications to start in dark mode automatically or by manually selecting the desired theme to check using the theme selector from the header or mobile menu.

## Checklist



### Required


### Appreciated


https://github.com/user-attachments/assets/ae577fc9-e567-49ed-8f70-c6aae8879e12
